SpaceShooter/SpaceShooter/CollisionManager.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Space_Shooter
+{
+    class CollisionManager
+    {
+        public List<Entity> Asteroids = new List<Entity>();
+        public List<Bullet> Bullets = new List<Bullet>();
+        public List<Planet> Planets = new List<Planet>();
```

Comment:
```suggestion
        public List<Entity> Asteroids = [];
        public List<Bullet> Bullets = [];
        public List<Planet> Planets = [];
```

Do you actually want these to be accessible publicly? If so, they should be wrapped in properties and abstracted in some way so that external code isn't overriding or modifying these in a way you don't intend